examples/metrics/run_as_cloud-run-job.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+if [[ -z "${GOOGLE_CLOUD_PROJECT}" ]]; then
+  echo "GOOGLE_CLOUD_PROJECT environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
+  echo "GOOGLE_APPLICATION_CREDENTIALS environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${GOOGLE_CLOUD_RUN_JOB_REGION}" ]]; then
+  echo "GOOGLE_CLOUD_RUN_JOB_REGION environment variable not set"
+  exit 1
+fi
+
+echo "ENVIRONMENT VARIABLES VERIFIED"
+
+echo "BUILDING SAMPLE APP IMAGE"
+gradle clean jib --image "gcr.io/${GOOGLE_CLOUD_PROJECT}/metrics-export-java"
+
+
+echo "CREATING A CLOUD RUN JOB TO RUN THE CONTAINER"
+gcloud run jobs create job-metrics-export \
+    --image "gcr.io/${GOOGLE_CLOUD_PROJECT}/metrics-export-java" \
```

Comment:
Container Registry is being deprecated in favor of Artifact Registry.  Should we update this?

Reply:
Agreed, but do you know if jib supports Artifact Registry ? I can only see [Google Container Registry](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#using-google-container-registry-gcr) being supported in the gradle plugin.

Reply:
Done